docs/icicle/primitives/overview.md

```diff
@@ -7,11 +7,3 @@ This section of the documentation is dedicated to the ICICLE primitives, we will
 
 
 - [MSM](./msm)
-- NTT
-    - Forward NTT
-    - Inverse NTT
```

Comment:
Why are we removing this?
We do support it but don't have documentation on it. I would add the NTT doc to the sidebar with a short one liner about we support it but the docs are in progress

Reply:
@omershlo requested me toremove it

Reply:
Ok, do we have tasks/plans to add full documentation for this or only once the new NTT is in?

Reply:
@jeremyfelder yes, i already have some branches for this